vhdx/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "vhdx"
+version = "0.1.0"
+authors = ["The Cloud Hypervisor Authors"]
+edition = "2018"
+license = "BSD-3-Clause"
+
+[dependencies]
+byteorder = "1.4.3"
+libc = "0.2.94"
+log = "0.4.14"
+remain = "0.2.2"
+vmm-sys-util = ">=0.3.1"
+crc32c = "0.6.0"
+uuid = { version = "0.8", features = ["serde", "v4"] }
```

Comment:
I don't think you need the `serde` feature for this crate, do you?
The latest version is `0.8.2`.